### PR TITLE
update external gallery

### PR DESCRIPTION
**Pull request by banesullivan**
Updates the external example gallery.

@akaszynski, can we land this before the release so that it gets deployed to the published docs?

### CODE REVIEW

File: docs/make_external_gallery.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+.. caution::
+
+    Please note that these examples link to external websites.
+    If any of these links are broken, please raise an issue on the repository.
+
+
+Do you have a technical processing workflow or visualization routine you
+would like to share?
+If so, please consider sharing your work here by clicking the `Edit on GitHub`
```
Comment (akaszynski) [inline docs/make_external_gallery.py:137]: We've removed the "Edit on GitHub" button for the time being.  Perhaps add "email us"?

**Comment by banesullivan** [comment]
Huh. From what I thought, dev.pyvista.org should only be the `master` branch between releases... not all branches.... another issue to add

**Comment by akaszynski** [comment]
Yea, that's going to change based on how we're doing releases too.  One more issue...

**Comment by akaszynski** [comment]
Merging so this will be with the PR release tomorrow.

**Pull request merged**